### PR TITLE
NAS-121719 / 23.10 / Add Zerotier to `community` train

### PR DESCRIPTION
**Pull request by stavros-k**

### CODE REVIEW

File: library/ix-dev/community/zerotier/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+annotations:
+  title: Zerotier
+type: application
+version: 1.0.0
+apiVersion: v2
+appVersion: '1.10.6'
+kubeVersion: '>=1.16.0-0'
+maintainers:
+  - name: truenas
+    url: https://www.truenas.com/
```
Comment (Qubad786) [inline library/ix-dev/community/zerotier/Chart.yaml:12]: Let's please add `email` attr as well

**Review by Qubad786 — COMMENTED** [review]
Let's please add the `metadata.yaml` file as well for all of these new apps which we are bringing in the open PRs

File: library/ix-dev/community/zerotier/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    description: Zerotier is able to set group ID for it's sub-processes.
+  - name: SETUID
+    description: Zerotier is able to set user ID for it's sub-processes.
+  - name: SETPCAP
+    description: Zerotier is able to set process capabilities.
+  - name: SYS_ADMIN
+    description: Zerotier is able to perform various system administration operations.
+hostMounts:
+  - hostPath: /dev/tun
+    description: Required to access the TUN device
```
Comment (Qubad786) [inline library/ix-dev/community/zerotier/metadata.yaml:32]: Probably better to re-word it to say why?

**Review by Qubad786 — APPROVED** [review]

File: library/ix-dev/community/zerotier/metadata.yaml (hunk 2 above)
Comment (stavros-k) [inline library/ix-dev/community/zerotier/metadata.yaml:32]:
```suggestion
    description: Requires access to the /dev/tun device in order to create the VPN tunnel.
```

**Pull request merged**